src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TO-DO List</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:ital,wght@1,300&display=swap" rel="stylesheet">
+    <script defer src="main.js"></script>
```

Comment:
[OPTIONAL] I think the best option for you right now to add Bootstrap to the project, is by simply adding the `cdn` link to the head of your file. Although this isn't the best alternative for production-ready projects, it is a really good alternative for light-weight projects.

You will find out as you advance in the program, that other tools come with webpack-ready builds where you can import libraries more easily.

If you still want to add bootstrap manually you could run `npm install bootstrap`, and then import `import 'bootstrap/dist/css/bootstrap.css'` to the top of your index.js file. (This is very similar to just importing the CDN script)

If you want more control over which parts of bootstrap you want in your project, you need to set up a pipeline to import SASS. [You can read more about here.](https://getbootstrap.com/docs/5.1/getting-started/download/) [And here.](https://github.com/twbs/bootstrap-npm-starter)